death-clock.js

```diff
@@ -36,6 +36,9 @@ let birthYear;
 let birthMonth;
 let birthDay;
 
+//doing this to only set it once for later on. Otherwise it fetch it every tick which seems wrong to me - Jet
+const siteurl = window.location;
```

Comment:
Let's change the variable name.

```suggestion
const siteUrl = window.location;
```